src/interactive/Tooltip.jsx

```diff
@@ -203,6 +209,9 @@ Tooltip.propTypes = {
 	trigger: PropTypes.element.isRequired,
 
 	/** The content that's rendered inside the tooltip */
+	children: PropTypes.element,
```

Comment:
I believe `element` enforces `children` to be a single element - is that the intent? I don't think there's a reason it can't be normal children, in which case I don't think you need to define it as a proptype

Reply:
No, there's no requirement to have one child.